test/unit/actions/proxy_action_test.rb

```diff
@@ -14,6 +14,9 @@ class ProxyActionTest <  ActiveSupport::TestCase
         Support::DummyProxyAction.any_instance.stubs(:with_batch_triggering?).returns(batch_triggering)
         Support::DummyProxyAction.reset
         RemoteTask.any_instance.stubs(:proxy).returns(Support::DummyProxyAction.proxy)
+        SettingRegistry.instance.stubs(:[]).with('foreman_tasks_proxy_action_retry_interval')
+        SettingRegistry.instance.stubs(:[]).with('foreman_tasks_proxy_action_retry_count')
+        SettingRegistry.instance.stubs(:[]).with('foreman_tasks_proxy_batch_trigger')
```

Comment:
Can we stub out `Setting` directly? we are reading it through `Setting` class, right?

Reply:
Correct, stubbed `Setting` directly